test/image/mocks/11.json

```diff
@@ -214,7 +214,8 @@
     "xatype": "category",
     "yatype": "linear",
     "tag": "",
-    "ref": "paper"
+    "xref": "paper",
+    "yref": "paper"
```

Comment:
just for my own learning, why has the `"ref"` property become `"xref"` and `"yref"`?

Reply:
So that you can place an annotation at a fixed horizontal position (`yref=“paper”`) and data-referenced vertical position (`xref=“x”`) for example. With a single `ref` things like that are not possible.